File: Assignment_012_Lb/program3.py
def Display(No1,No2):
    i = 0
    j = 0

    for i in range(1,No1 + 1,1):
        for j in range(No2,0,-1):
            print(j,end="\t")
        print()


def main():
    print("Enter Number of Rows :")
    Value1 = int(input())

    print("Enter Number of Columb :")
    Value2 = int(input())

    Display(Value1,Value2)

if __name__ == "__main__":
    main()

File: Assignment_012_Lb/program3.c
#include <stdio.h>

void DisplayPattern(int iNo1,int iNo2)
{
    int i = 0, j = 0;
    for(i = 1;i <= iNo1;i++)
    {
        for(j = iNo2; j >= 1;j--)
        {
            printf("%d\t",j);
        }
        printf("\n");
    }
}

int main()
{
    int iValue1 = 0,iValue2 = 0;

    printf("Enter Number of Rows and Columb:");
    scanf("%d %d",&iValue1,&iValue2);

    DisplayPattern(iValue1,iValue2);

    return 0;
}